src/TestCheck/EveryTestHasSameNamespaceAsTestedClass.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cdn77\TestUtils\TestCheck;
+
+use Cdn77\EntityFqnExtractor\ClassExtractor;
+use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+use function class_exists;
+use function Safe\preg_match;
+use function Safe\sprintf;
+use function Safe\substr;
+use function str_replace;
+use function trait_exists;
+
+final class EveryTestHasSameNamespaceAsTestedClass implements TestCheck
+{
+    private const PATTERN = '~\* @testedClass (?<targetClass>.+)\n~';
+
+    /** @var iterable<string> $filePathNames */
+    private $filePathNames;
+
+    /** @var TestCase */
+    private $testCaseContext;
+
+    /** @param iterable<string> $filePathNames */
+    public function __construct(TestCase $testCaseContext, iterable $filePathNames)
+    {
+        $this->filePathNames = $filePathNames;
+        $this->testCaseContext = $testCaseContext;
+    }
+
+    public function run() : void
+    {
+        $this->testCaseContext::assertTrue(true);
```

Comment:
proč tohle?

Reply:
coverage a bergman, kterej nepocita coverage, ale assertcoverage

Reply:
nechápu ... afaik existuje tag pro testy co nedělají asserty, to by to mělo řešit

Reply:
neresi

Reply:
ok už jsem našel tu issue,a nejspíš jsme to už jednou řešili lol ... no, mrknu na phpunit alternativy lol, Bergmana začínám mít docela dost už :D